OpenEdXMobile/res/layout/activity_launch.xml

```diff
@@ -6,9 +6,16 @@
         android:id="@+id/rl_root"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        tools:background="@drawable/launch_background"
```

Comment:
@adamsythe tools:background="@drawable/launch_background" used only for editor view.

Reply:
Great are you waiting on me for anything